main.py

```diff
@@ -490,7 +490,53 @@ def index(self):
             p.type = point.type
             points.append(p)
         context = {'points': points}
-        return self.render('viewhighlighted.html', **context)
+        return self.render('viewhighlighteddata.html', **context)
+
+    def is_visible(self):
+        return login.current_user.is_authenticated()
+
+class ViewHighlightedMarkersMap(BaseView):
```

Comment:
O.K, for now I changed it so the admin view calls index itself but we will have to adjust the view later so it will fit the admin layout.